### PR TITLE
Update `osu! originals` with 11 tracks

### PR DESCRIPTION
**Pull request by Walavouchey**
<image width="60%" src="https://github.com/ppy/osu-wiki/assets/36758269/d053eb71-2bde-4515-85ee-fb25784cd72f">

### CODE REVIEW

File: wiki/osu!_originals/es.md [hunk 1]
```diff
@@ -623,6 +641,15 @@ Estas canciones se [anunciaron](https://osu.ppy.sh/home/news/2023-02-01-twin-tri
...
 | [takehirotei - See you next time!](https://soundcloud.com/takehirotei/see-you-next-time) |  | Canción de los créditos |
 
+#### ![](/wiki/shared/mode/mania.png) osu!mania 4K Chinese National Cup 2023
+
+| Canción | Beatmap | Notas |
+| :-: | :-: | :-- |
+| dMinor117 & ShirinNeko - d Minor Sonatine (Insomnia) | [#1](https://osu.ppy.sh/beatmapsets/2019496) | Playoff 2 RC5 |
+| Laxeno57 - Kaleido | [#1](https://osu.ppy.sh/beatmapsets/2043910) | Desempate de la gran final |
+| [SAMString - Celestial Solace](https://soundcloud.com/samstring/celestial-solace)[^fa] | [#1](https://osu.ppy.sh/beatmapsets/2031765) | Cuartos de final HB2 |
+| [Sydosys - she grand on my finals till i hb3](https://soundcloud.com/sydosys/she-grand-on-my-finals-till-i-hb3) | [#1](https://osu.ppy.sh/beatmapsets/2043993) | HB3 de la gran final |
```
Comment (agatemosu) [inline wiki/osu!_originals/es.md:651]:
```suggestion
| [Sydosys - she grand on my finals till i hb3](https://soundcloud.com/sydosys/she-grand-on-my-finals-till-i-hb3) | [#1](https://osu.ppy.sh/beatmapsets/2043993) | Gran final HB3 |
```

File: wiki/osu!_originals/es.md [hunk 2]
```diff
@@ -739,6 +778,12 @@ Estas canciones se [anunciaron](https://osu.ppy.sh/home/news/2023-02-01-twin-tri
 | [blobdash & breakchild - EPiSODES](https://soundcloud.com/blobdash/episodes) | [#1](https://osu.ppy.sh/beatmapsets/1901573) | Desempate de la gran final |
 | [Sephid - Hypothesize](https://www.youtube.com/watch?v=FvaJrO3wJbI)[^fa] | [#1](https://osu.ppy.sh/beatmapsets/1897397) | Desempate de la final |
 
+#### ![](/wiki/shared/mode/osu.png) Scarlet's Touhou Tournament 3rd Season
+
+| Canción | Beatmap | Notas |
+| :-: | :-: | :-- |
+| nominomu - solar sect of nm2 ~ nuclear notelock | [#1](https://osu.ppy.sh/beatmapsets/2039737) | NM2 de octavos de final |
```
Comment (agatemosu) [inline wiki/osu!_originals/es.md:785]:
```suggestion
| nominomu - solar sect of nm2 ~ nuclear notelock | [#1](https://osu.ppy.sh/beatmapsets/2039737) | Octavos de final NM2 |
```

File: wiki/osu!_originals/es.md [hunk 3]
```diff
@@ -623,6 +641,15 @@ Estas canciones se [anunciaron](https://osu.ppy.sh/home/news/2023-02-01-twin-tri
 | [takehirotei as "Infinite Limit" - C:// YOU NEX-TIME](https://soundcloud.com/takehirotei/takehirotei-as-infinite-limit-see-you-next-time) | [#1](https://osu.ppy.sh/beatmapsets/2028305) | Desempate de la gran final |
 | [takehirotei - See you next time!](https://soundcloud.com/takehirotei/see-you-next-time) |  | Canción de los créditos |
 
+#### ![](/wiki/shared/mode/mania.png) osu!mania 4K Chinese National Cup 2023
+
+| Canción | Beatmap | Notas |
+| :-: | :-: | :-- |
+| dMinor117 & ShirinNeko - d Minor Sonatine (Insomnia) | [#1](https://osu.ppy.sh/beatmapsets/2019496) | Playoff 2 RC5 |
```
Comment (agatemosu) [inline wiki/osu!_originals/es.md:648]:
```suggestion
| dMinor117 & ShirinNeko - d Minor Sonatine (Insomnia) | [#1](https://osu.ppy.sh/beatmapsets/2019496) | Eliminatorias 2 RC5 |
```

File: wiki/osu!_originals/es.md [hunk 4]
```diff
@@ -329,6 +341,12 @@ Estas canciones se [anunciaron](https://osu.ppy.sh/home/news/2023-02-01-twin-tri
 | :-: | :-: | :-- |
 | Daily197 - Sirius | [#1](https://osu.ppy.sh/beatmapsets/1964081) | Desempate de la gran final |
 
+#### ![](/wiki/shared/mode/osu.png) Battle of the Box 2023: Trouble in Boxville
+
+| Canción | Beatmap | Notas |
+| :-: | :-: | :-- |
+| [ryo (supercell) feat. Hatsune Miku - Kocchi Muite Baby (Sydosys remix)](https://soundcloud.com/sydosys/ryo-supercell-feat-hatsune-miku-kocchi-muite-baby-sydosys-remix) | [#1](https://osu.ppy.sh/beatmapsets/1968676) | HD3 de la gran final |
```
Comment (agatemosu) [inline wiki/osu!_originals/es.md:348]:
```suggestion
| [ryo (supercell) feat. Hatsune Miku - Kocchi Muite Baby (Sydosys remix)](https://soundcloud.com/sydosys/ryo-supercell-feat-hatsune-miku-kocchi-muite-baby-sydosys-remix) | [#1](https://osu.ppy.sh/beatmapsets/1968676) | Gran final HD3 |
```

File: wiki/osu!_originals/fr.md [hunk 5]
```diff
@@ -623,6 +641,15 @@ Ces musiques ont été [annoncées](https://osu.ppy.sh/home/news/2023-02-01-twin
 | [takehirotei as "Infinite Limit" - C:// YOU NEX-TIME](https://soundcloud.com/takehirotei/takehirotei-as-infinite-limit-see-you-next-time) | [#1](https://osu.ppy.sh/beatmapsets/2028305) | Tiebreaker des Grandes Finales |
 | [takehirotei - See you next time!](https://soundcloud.com/takehirotei/see-you-next-time) |  | Thème du générique de fin |
 
+#### ![](/wiki/shared/mode/mania.png) osu!mania 4K Chinese National Cup 2023
+
+| Musique | Beatmap | Notes |
+| :-: | :-: | :-- |
+| dMinor117 & ShirinNeko - d Minor Sonatine (Insomnia) | [#1](https://osu.ppy.sh/beatmapsets/2019496) | Playoff 2 RC5 |
```
Comment (Walavouchey) [inline wiki/osu!_originals/fr.md:648]: any french reviewer for this one? (@DoryanR)

Comment (DoryanR) [reply wiki/osu!_originals/fr.md:648]: It's good for me 👍

**Review by cl8n — APPROVED** [review]

**Pull request merged**